apps/base-docs/tutorials/docs/1_event-gate-nouns.md

```diff
@@ -0,0 +1,429 @@
+---
+title: 'Gate IRL Events with Nouns'
+slug: /event-gate-with-nouns
+description: Learn how to gate entry to an IRL event for members of a Nounish DAO.
+author: briandoyle81
+keywords: [Solidity, ERC-721, token, NFT, Gate, Nouns, PFP]
+tags: ['nft', 'smart contracts', 'frontend']
+difficulty: intermediate
+hide_table_of_contents: false
+displayed_sidebar: null
+---
+
+You've probably seen people in onchain communities use ⌐◨-◨ in their profile names. These are called _Nouns Goggles_, or _Noggles_. They're an ASCII representation of the glasses found on every procedurally generated [Nouns] NFT avatar. The [Nouns Auction] makes one new Noun available for auction every single day - forever!
+
```

Comment:
Really appreciate you adding Nouns lore to this tutorials. I think this is very crypto forward in a cultural (as apose to technical) way.